src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -62,13 +62,13 @@ struct GatherSlice
         }
 
         int countRanks = Environment<simDim>::get().GridController().getGpuNodes().productOfComponents();
-        int gatherRanks[countRanks];
-        int groupRanks[countRanks];
+        std::vector<int> gatherRanks(countRanks);
+        std::vector<int> groupRanks(countRanks);
         mpiRank = Environment<simDim>::get().GridController().getGlobalRank();
         if (!isActive)
             mpiRank = -1;
 
-        MPI_CHECK(MPI_Allgather(&mpiRank, 1, MPI_INT, gatherRanks, 1, MPI_INT, MPI_COMM_WORLD));
+        MPI_CHECK(MPI_Allgather(&mpiRank, 1, MPI_INT, gatherRanks.data(), 1, MPI_INT, MPI_COMM_WORLD));
```

Comment:
`std::vector<...>.data` is C++11 :-( only. Currently our code not use C++11 features.